### PR TITLE
chore: add temp code replacement hack due to gapic code-gen template bug

### PR DESCRIPTION
**Pull request by stephaniewang526**
Depends on https://github.com/googleapis/gapic-generator/issues/3181

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -34,4 +34,7 @@
       destination_name='bigquerystorage',
   )
 
-java.common_templates()
+java.common_templates(excludes=[
+    #TODO: REMOVE THIS WHEN GRPC FIXES ITS TEMPLATE
+    'google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/MockBigQueryWriteImpl.java'
```
Comment (chingor13) [inline synth.py:39]: This file is not a common template, so this won't work. The client copying happens in the `java.bazel_library` command above and does not include an option for excluding files.

To fix in synth, you will want to use `s.replace(files, search_regex, replace_string)`

Comment (stephaniewang526) [reply synth.py]: Done.

File: synth.py [hunk 2]
```diff
@@ -35,3 +35,23 @@
   )
 
 java.common_templates()
+
+# TODO: Remove all below s.replace() logic when upstream correction is made in gapic
```
Comment (chingor13) [inline synth.py:39]: Please link to the upstream issue on gapic-generator

File: synth.py [hunk 3]
```diff
@@ -35,3 +35,23 @@
...
+          public void onNext(AppendRowsRequest value) {
+            if (response instanceof AppendRowsResponse) {
+          """,
+          """
+          public void onNext(AppendRowsRequest value) {
+            requests.add(value);
+            final Object response = responses.remove();
+            if (response instanceof AppendRowsResponse) {
+          """
+          )
```
Comment (chingor13) [inline synth.py:57]: You may want to re-run the java formatter after doing the code replacements.

File: synth.py (hunk 2 above)
Comment (stephaniewang526) [reply synth.py]: Done.

File: synth.py (hunk 3 above)
Comment (stephaniewang526) [reply synth.py]: Done.

**Pull request merged**